### ISSUE TITLE
[Disocrd becoming laggy-crashes] - DisplayServersAsChannels

### ISSUE THREAD

**Issue by alexandrejbc** [mwittrien/BetterDiscordAddons#763]
When this plugin is on, whenever i go in fullscreen on discord call and that i get back to normal screen discord becomes laggy (crash sometimes), it has like a 15fps, and the discord servers list disappears.
It occurs whenever i go in fullscreen, I don't know yet if there is other way to the bug to occur

**Expected behavior**
I guess i wanted to not be laggy whenever i go fullscreen on a call haha

**Screenshots**
https://youtu.be/VPjMjZ7_xaM
basically what's happening

**Discord Build (stable, ptb or canary):**
[BUILD INFO] Release Channel: stable, Build Number: 82866, Version Hash: d02d7ca5a6ae4aea042644370e7868bbec8c51d7

for now i'll just turn it off but wp guys it's really cool imo it looks rly smooth
